### DISCUSSION TITLE
Make the layout of notebook template to be consistent

### DISCUSSION THREAD

**Discussion by wycc** [wycc/cgu-kubeflow#3] [General]
![image](https://github.com/wycc/cgu-kubeflow/assets/457473/ab6a149b-4b1e-4ce4-8ce0-16fc75f2696e)

**Comment by wycc** [comment]
(1) Let the size of each box consistent
(2) Put the button in the end of the box

**Reply by wycc** [reply]
Add an extra search field

**Reply by wycc** [reply]
![image](https://github.com/wycc/cgu-kubeflow/assets/457473/8baf76a4-021c-41ee-901b-25d9b8e728c8)
 scroll bar move to the grid list only. DO not scroll the name and namesapce.

**Reply by lance-cl-lu** [reply]
YC, Roger that
